src/lib/ui/WcProductCard.svelte

```
@@ -20,6 +28,49 @@ Wraps the <product-card> web component and adds accessibility features.
+		if (p.brands_tags && p.brands_tags.length > 0) {
+			getTaxo<Brand>('brands', fetch)
+				.then((t) => (brandsTaxonomy = t))
+				.catch(console.error);
+		}
+	});
+
+	let sanitizedProduct = $derived.by(() => {
+		const p = product as Product;
+		const taxo = brandsTaxonomy;
+		let brandsTags = p.brands_tags;
+		let brandsStr = p.brands;
```

ESLint

[error] The value assigned to 'brandsStr' is not used in subsequent statements.

(no-useless-assignment)